packages/ap-frontend/alliance_platform/frontend/bundler/vite.py

```diff
@@ -245,6 +245,16 @@ def __init__(self, root_dir: Path, manifest_file: Path):
             # will resolve to components/table/index.tsx
             if key.stem == "index":
                 entries[key.parent] = entries[key]
+
+            # This "unresolvedPath" is an extension we provide with a custom plugin. This stores the unresolved path
```

Comment:
Since this is currently a `platform-py` specific section that relies on something in the template project, I think it would be good to add a couple of notes: one specifying to look in the template project for the corresponding code, and another with a TODO to reflect the intention that Vite functionality be moved to a separate package at some point.

Reply:
Done